aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -195,6 +195,10 @@ Resources:
           - Effect: Allow
             Action: 'sagemaker:InvokeEndpoint'
             Resource: !Sub "arn:aws:sagemaker:${AWS::Region}:${AWS::AccountId}:endpoint/gen-ai-*"
+          # Comprehend Detect Toxic Content access
+          - Effect: Allow
+            Action: 'comprehend:DetectToxicContent'
+            Resource: '*'
```

Comment:
Do we want to restrict to `gen-ai-*` as we do for sagemaker?

Reply:
The `gen-ai-*` resource restriction for sagemaker `InvokeEndpoint` restricts access to endpoints that begin with `gen-ai-`. For Comprehend `DetectToxicContent`, there isn't a specific resource we're targeting so I think we have to use a wildcard (see https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncomprehend.html#amazoncomprehend-resources-for-iam-policies)